app/jobs/heartbeat_job.rb

```diff
@@ -2,7 +2,7 @@ class HeartbeatJob < ApplicationJob
   queue_as :default
 
   def perform
-    Rails.logger.info(
+    IdentityJobLogSubscriber.new.logger.info(
```

Comment:
Every time we instantiate a new logger I think it prints the same `# generated by Logger` preamble, would it make sense to have a static one of these loggers hanging around anywhere?